index.html

```diff
@@ -36,7 +36,7 @@
         </nav>
         
         <!--List members bar-->
-        <div class="container-fluid" id="listMembersBar">
```

Comment:
It's usually better to use classes instead of ID's because ID's are supposed to be completely unique and it's hard to make sure you don't mess it up. (In fact, as you can see, there were two places where you had `id="listMembersBar"` which is technically not allowed, though most browsers will be nice and not completely blow up the page)